project_explainer_ui/streamlit_ui.py

```diff
@@ -0,0 +1,29 @@
+from gh_explainer import Explainer
+
+def summarize(summarization_type, github_project_url, github_project_branch="main", huggingface_model_id="gpt2"):
+    gptExplainer = Explainer(huggingface_model_id)
+    if summarization_type == "brief":
+        return gptExplainer.brief(github_url=github_project_url, branch=github_project_branch)["summary"]
+    return gptExplainer.outline(github_url=github_project_url, branch=github_project_branch)["summary"]
+
+
+import streamlit as st
```

Comment:
Can we move imports to top cleanly?

Reply:
Sorry, It was my mistake!,I will make it clean